ocs/pkg/service/v0/users.go

```diff
@@ -358,6 +367,100 @@ func (o Ocs) DeleteUser(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if o.config.RevaAddress != "" && o.config.StorageUsersDriver != "owncloud" {
```

Comment:
Why does this not work for the `owncloud` driver?

Reply:
@butonic https://github.com/cs3org/reva/blob/master/pkg/storage/fs/owncloud/owncloud.go#L1146-L1152

```go
// If home is enabled, the relative home is always the empty string
func (fs *ocfs) GetHome(ctx context.Context) (string, error) {
	if !fs.c.EnableHome {
		return "", errtypes.NotSupported("ocfs: get home not supported")
	}
	return "", nil
}
```

Reply:
the logic probably should be more generic, but this at least makes the only consumer (CI) happy